cms/src/components/languages/CreateOrEditLanguageModel.spec.ts

```diff
@@ -80,22 +90,6 @@ describe("CreateOrEditLanguageModal.vue", () => {
             // check if the button has the right text
             expect(wrapper.html()).toContain("Save Changes");
         });
-
-        it("disables save button if form fields are not filled", async () => {
```

Comment:
Just realised this test was not a duplicate, this test tests if the button becomes disables if all the other values are empty, so you need to check if disabled is truthy